packages/twenty-front/src/modules/object-record/record-right-drawer/components/RightDrawerRecord.tsx

```diff
@@ -6,6 +6,13 @@ import { RecordShowContainer } from '@/object-record/record-show/components/Reco
 import { useRecordShowPage } from '@/object-record/record-show/hooks/useRecordShowPage';
 import { RecordValueSetterEffect } from '@/object-record/record-store/components/RecordValueSetterEffect';
 import { RecordFieldValueSelectorContextProvider } from '@/object-record/record-store/contexts/RecordFieldValueSelectorContext';
+import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
+import styled from '@emotion/styled';
+
+const StyledRightDrawerRecord = styled.div`
+  height: ${({ theme }) =>
+    useIsMobile() ? `calc(100% - ${theme.spacing(16)})` : '100%'};
```

Comment:
@bosiraphael 
This is correct right? this is regarding the height of the right drawer. 
Should i just use a local const like in CommandMenu? or create new const file and export it from there to both the files

Reply:
Second option is better to avoid code duplication :)